CHANGELOG.md

```diff
@@ -186,28 +185,28 @@ Updated the extension to be compatible with the latest version of Visual Studio
 
     ![alt_text](images/improved-logs-viewer_1_2_0.gif "Improved logs viewer")
 
-### Bug fixes
+### Bug Fixes
 
-* [Issue #176:](https://github.com/GoogleCloudPlatform/cloud-code-vscode/issues/176) Cloud Code will no longer continually switch user context to the output window!
-* [Issue #201:](https://github.com/GoogleCloudPlatform/cloud-code-vscode/issues/201) Fixed an issue where the Logs Viewer hangs when you don’t have permission to list all namespaces.
+* Cloud Code will no longer continually switch user context to the output window. [#176](https://github.com/GoogleCloudPlatform/cloud-code-vscode/issues/176) 
+* Fixed an issue where the Logs Viewer hangs when you don’t have permission to list all namespaces. [#201](https://github.com/GoogleCloudPlatform/cloud-code-vscode/issues/201) 
 * Fixed an issue where Open with Cloud Code would not clone a repository in some cases.
 * Fixed an issue where Cloud Shell would fail to create an SSH configuration when the gcloud command-line tool prompted for survey participation.
 
-## Version 1.1.2
+## Version 1.1.2: Feb 27, 2020
```

Comment:
```suggestion
## Version 1.1.2 (Feb 27, 2020)
```

Reply:
Good catch! missing string replaces, updated the rest of them as well.